src/index.ts

```diff
@@ -20,6 +20,10 @@ const log = getLogger(__filename);
 // Setup cache storage
 export let cacheStorage: storage.StorageService;
```

Comment:
This variable at module level will cause problems if multiple clients are instantiated. Please manage this variable inside the local state of the client.